Miki.Framework.Commands.Localization/Miki.Framework.Commands.Localization.csproj

```diff
@@ -12,4 +13,10 @@
     <ProjectReference Include="..\Miki.Framework.Commands\Miki.Framework.Commands.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.Threading.Tasks.Extensions">
+      <HintPath>C:\Users\Veld\.nuget\packages\system.threading.tasks.extensions\4.5.0\ref\netstandard2.0\System.Threading.Tasks.Extensions.dll</HintPath>
```

Comment:
I think this should be a NuGet package reference instead of a direct DLL reference 😉.